tests/run-make/extern-alias/Makefile

```diff
@@ -0,0 +1,11 @@
+# ignore-cross-compile
+include ../tools.mk
+
+all:
+	$(RUSTC) --crate-type=rlib unwieldy_crate_name.rs -o $(TMPDIR)/libunwieldy_crate_name.rlib
+	$(RUSTC) --crate-type rlib main.rs --extern unwieldy_crate_name=$(TMPDIR)/libunwieldy_crate_name.rlib \
+		2>&1 | $(CGREP) "use nice_crate_name::Foo;"
+
+	$(RUSTC) --crate-type=rlib --edition=2021 unwieldy_crate_name.rs -o $(TMPDIR)/libunwieldy_crate_name.rlib
+	$(RUSTC) --crate-type rlib main.rs --edition=2021 --extern unwieldy_crate_name=$(TMPDIR)/libunwieldy_crate_name.rlib \
+		2>&1 | $(CGREP) "use crate::nice_crate_name::Foo;"
```

Comment:
Does this need to be `run-make` tests? There are ui tests which  use `--extern` that you might be able to base this on.